src/targets/chrome/get-selector-box-size.js

```diff
@@ -12,6 +12,27 @@ const getSelectorBoxSize = (window, selector) => {
     return !isWrapper;
   };
 
+  const isVisisble = element => {
+    let style = {};
+    try {
+      style = window.getComputedStyle(element);
```

Comment:
When would this throw?

Reply:
It would throw when `element` is not an instance of `Element`

Reply:
IDK, feels like that shouldn't happen and that it should really throw in that case.

Reply:
Done! ✅